aten/src/ATen/native/TensorCompare.cpp

```diff
@@ -289,6 +291,27 @@ Tensor _s_where(const Tensor& condition, const Tensor& self, const Tensor& other
   return ret;
 }
 
+void zero_numel_tensor_resize(Tensor& result, Tensor& result_indices, const Tensor& self, int64_t dim,
```

Comment:
There isn't really any reason to take result by mutable reference here, can it.

Reply:
There is a call to `resize_` in the function for resizing the result so it should be mutable, no?

Reply:
If you take a quick peek, you'll find that `resize_` method is const-qualified :) The mutable references are a lie and they allow for mistakes (`result = ...`, which is ALWAYS ALWAYS wrong)

Reply:
Ah I see. OK will change it then.